src/dlgRoomExits.cpp

```diff
@@ -284,7 +284,7 @@ void dlgRoomExits::save()
             door = 0;
         }
         QString value = pI->text(7);
-        if (value != tr("<command or Lua script>", "This string is also used programmatically ensure all instances are the same, (3 of 5)") && key != 0 && mpHost->mpMap->mpRoomDB->getRoom(key) != 0) {
+        if (value != tr("<command or Lua script>", "This string is also used programmatically ensure all instances are the same, (3 of 5)") && key != 0 && mpHost->mpMap->mpRoomDB->getRoom(key) != nullptr) {
```

Comment:
In the same manner in `key != 0 && mpHost->mpMap->mpRoomDB->getRoom(key) != nullptr` the `!= 0` and the `!= nullptr` are also superfluous.